src/main/java/uk/gov/companieshouse/officerfiling/api/model/entity/OfficerFilingData.java

```diff
@@ -270,6 +282,8 @@ public String toString() {
                 .add("residentialAddressSameAsCorrespondenceAddress="
                         + residentialAddressSameAsCorrespondenceAddress)
                 .add("corporateDirector=" + corporateDirector)
+                .add("whereDirectorLiveChoice=" + directorResidentialAddressChoice)
```

Comment:
should we rename to string to reflect the variables ?

Reply:
Renamed